.github/workflows/lint.yml

```diff
@@ -0,0 +1,26 @@
+name: Lint
+
+on:
+  push:
+    paths:
+      - '*.py'
+
+jobs:
+  flake8_py3:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup Python
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.8
+          architecture: x64
+      - name: Checkout master branch
+        uses: actions/checkout@master
+      - name: Install flake8
+        run: pip install flake8
```

Comment:
You need to install all our flake8 plugins too, preferably with version pinning included. Consider creating a `lint-requirements.txt` file and making `setup.py` read its contents. Or you could just install the entire project with the dev extras like we used to do.

Reply:
Fixed in https://github.com/python-discord/django-crispy-bulma/pull/34/commits/13d70c41ef7f15cfccf023d151d6440bd3ebb3a3